vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1597,6 +1601,17 @@ class TimelineFragment @Inject constructor(
         voiceMessageRecorderView.isVisible = false
     }
 
+    private fun renderTypingMessageNotification(roomSummary: RoomSummary?, state: RoomDetailViewState) {
+        if (!isThreadTimeLine() && roomSummary != null) {
+            views.typingMessageView.isInvisible = state.typingUsers.isNullOrEmpty()
```

Comment:
So I understand than when there is no typing user, there is an empty area at the bottom of the timeline, right?

Reply:
Yes, it's about 20dp, otherwise we will see the jump of the time line recycler view.

Reply:
OK, thanks